patroni/dcs.py

```diff
@@ -216,8 +244,29 @@ def attempt_to_acquire_leader(self):
     def set_failover_value(self, value, index=None):
         """Create or update `/failover` key"""
 
-    def manual_failover(self, leader, member, index=None):
-        return self.set_failover_value(leader + (':' + member if member else ''), index)
+    def manual_failover(self, leader, member, planned_at=None, index=None):
```

Comment:
Looks like all invocations of this method already do the parse/localize time conversion, so there is no need to call it agin inside.